layerInfo.json

```diff
@@ -66,7 +66,7 @@
 					"Core-Interfaces",
 					"Core-Utils",
 					"Client-Utils",
-					"Protocol-Definitions"
+					"Driver-Definitions"
```

Comment:
Telemetry-Utils now depends on driver-definitions instead of protocol-definitions.